src/content/enterprise/install/upgrade.mdx

```diff
@@ -29,8 +17,97 @@ $ helm upgrade <your-release-name> okteto/okteto -f config.yaml --namespace=okte
 For example:
 ```console
 $ helm repo update
-$ helm upgrade enterprise okteto/okteto -f config.yaml --namespace=okteto --version 0.9.x
+$ helm upgrade okteto okteto/okteto -f config.yaml --namespace=okteto --version 0.9.x
 ```
 *You can use `helm ls` to find the name of your release.*
 
 Please review [the release notes](/docs/enterprise/install/releases/) before upgrading. New features, known issues, and configuration changes will be listed there.
+
+## Upgrading to 1.x
+
+The Okteto chart 1.0 comes with two breaking changes due to the following changes:
```

Comment:
Let's be consistent in how we call it. We use ' Okteto chart 1.0', 'chart', and 'helm chart' across the doc.